### ISSUE TITLE
image paint

### ISSUE THREAD

**Issue by lijain** [fenglinglwb/MAT#87]
Hello, may I ask why the eyes and mouths of some animals are somewhat distorted when I perform finetune on my data set and use a small mask to repair images of the same size as 512? Is there a problem with my Settings?
![image](https://github.com/fenglinglwb/MAT/assets/18046568/cfb9e17f-9268-4efc-af2a-6a8cbe1d77e7)

![image](https://github.com/fenglinglwb/MAT/assets/18046568/78508f52-d671-4478-b47a-f5a14741c77f)

**Comment by fenglinglwb** [comment]
The model pre-trained on Places sees scenes rather than individual objects. You could prolong the fine-tuning process.

**Comment by lijain** [comment]
Thank you for your patient answer. There are many types of data in this section, not only scenes but also animals. Data 60w I've fine-tuned 17 rounds

**Comment by lijain** [comment]
Hello, I wonder if you can explain? Is it because animals have different eyes than humans?

**Comment by fenglinglwb** [comment]
Sure, there is almost no such data in Places. You may train from scratch to see whether this helps.

**Comment by sofpya** [comment]
Hello, I am also trying to complement the animal's face using this code. Have you solved this problme?
